### DISCUSSION TITLE
Sorttitle

### DISCUSSION THREAD

**Discussion by Bartyboy** [croneter/PlexKodiConnect#1764] [Q&A] [answered]
Hi
Can you tell me where PKC takes out the information for the sort order for titles?
I have a movie named 'the amazing spiderman'. On Plex the title and sorttitle is 'the amazing spiderman. the same thing on the collection.
PKC sorts it under 'a' for 'amazing spiderman'.
I've deleted the database and made it new but there is no change.

**Comment by croneter** [comment, marked as answer]
PlexKodiConnect does not do anything, Kodi is responsible for that. See Ignore articles when sorting here: http://kodi.wiki/view/Settings/Media/General

**Reply by Bartyboy** [reply]
Thank you very much! I searched and tested a few things but I didn' saw this point.
Now all is sorted perfectly!
